pages/resources/quests.mdx

```diff
@@ -30,43 +30,50 @@ Represents a sponsored quest.
-| assignment_method | integer                                               | Unknown integer that controls something related to reward assignment |
-| rewards           | array[[quest reward](#quest-reward-structure) object] | Which rewards the user receives upon completing the quest            |
+| Field             | Type                                                  | Description                                                                                             |
+| ----------------- | ----------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| assignment_method | integer                                               | [An enum that determines which method gets used to asisgn the rewards](#quest-reward-assignment-method) |
+| rewards           | array[[quest reward](#quest-reward-structure) object] | Which rewards the user receives upon completing the quest                                               |
 
 ###### Quest Reward Structure
 
-| Field    | Type                                                             | Description                                                                   |
-| -------- | ---------------------------------------------------------------- | ----------------------------------------------------------------------------- |
-| sku_id   | snowflake                                                        | Snowflake that specifies the collectible to award upon the quest's completion |
-| tag      | integer                                                          | The [reward type](#quest-reward-type)                                         |
-| messages | [quest reward messages](#quest-reward-messages-structure) object | Defines human-readable quest metadata                                         |
+| Field                 | Type                                                             | Description                                                 |
+| --------------------- | ---------------------------------------------------------------- | ----------------------------------------------------------- |
+| sku_id                | snowflake                                                        | The reward's ID                                             |
+| tag                   | integer                                                          | The [reward's type](#quest-reward-type)                     |
+| messages              | [quest reward messages](#quest-reward-messages-structure) object | Human-readable metadata for the reward                      |
+| asset                 | ?string                                                          | The reward's media asset                                    |
+| approximate_count ^1^ | ?integer                                                         | An approximate count of how many users can claim the reward |
```

Comment:
```suggestion
| asset?                 | string                                                          | The reward's media asset                                    |
| approximate_count? ^1^ | integer                                                         | An approximate count of how many users can claim the reward |
```
These fields are optional but not nullable, therefore the `?` goes next to the field name (see [this](https://docs.discord.sex/reference#example-nullable-and-optional-fields)).
If they were always present but sometimes set to `null` then it would be written like you did, but in this case it's the opposite.